lib/src/clojure_lsp/feature/hover.clj

```diff
@@ -65,6 +65,37 @@
                              (map #(str %))
                              (string/join "\n---\n"))))))
 
+(defn recurse-doc-line
```

Comment:
A little bit scarry this function, besides the test, maybe add a docstring why we want to do that with a minimal example?

Reply:
Done, and I've renamed it to be more clear about intent.